phold/README.md

```diff
@@ -1,7 +1,121 @@
 This directory contains code for a PHOLD benchmark implemented using SST. 
+The PHOLD benchmark is based on Fujimoto's 1990 paper [Performance of Time Warp Under Synthetic Workloads](https://gdo149.llnl.gov/attachments/20776356/24674621.pdf).
+PHOLD is a synthetic benchmark developed to surface important simulation characteristics as parameters. 
+This makes it useful for studying the behavior of PDES simulators like SST.
+
+## Prerequisites
+
+It is assumed you have installed [SST version 15.1.0](https://sst-simulator.org/SSTPages/SSTMainDownloads/). 
+You can check which version of `sst` you have installed with the following command: `sst --version`.
+
```

Comment:
Please include a sentence about how they can run PHOLD using containers and point to Ahmad's README about that.